CHANGES.md

```diff
@@ -1,5 +1,32 @@
 # Changes
 
+## edge-21.5.1
+
+This edge release adds support for versioned hint URLs in `linkerd check` and
+support for traffic splitting through ServiceProfiles, among other fixes and
+improvements. Additionally, support has also been introduced for `NodePort`
+type services to the multicluster helm chart.
+
+
```

Comment:
This line is what the linter is complaining about :wink: 
Hopefully the other int tests failures are flakes :pray: